### PR TITLE
Add test to all django admin views

### PR DESCRIPTION
**Pull request by szok**
Add test to all django admin views:
Tested:
- number of SQL queries
- if all views respond to correct status

### CODE REVIEW

File: src/ralph/admin/tests/tests_views.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            model_class_path = '{}.{}'.format(model.__module__, model.__name__)
+            if model_class_path in EXCLUDE_MODELS:
+                continue
+            print(model_class_path)
+            factory = FACTORY_MAP.get(model_class_path)
+            module_path, factory_class = factory.rsplit('.', 1)
+            module = import_module(module_path)
+            factory_model = getattr(module, factory_class)
+            # Create 10 records:
+            for _ in range(10):
```
Comment (mkurek) [inline src/ralph/admin/tests/tests_views.py]: use https://factoryboy.readthedocs.io/en/latest/reference.html#factory.Factory.create_batch

File: src/ralph/admin/tests/tests_views.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        self.request.session = {}
+
+    def test_numbers_of_sql_query_and_response_status_is_200(self):
+        for model, model_admin in ralph_site._registry.items():
+            query_count = 0
+            model_class_path = '{}.{}'.format(model.__module__, model.__name__)
+            if model_class_path in EXCLUDE_MODELS:
+                continue
+            print(model_class_path)
+            factory = FACTORY_MAP.get(model_class_path)
```
Comment (mkurek) [inline src/ralph/admin/tests/tests_views.py]: FACTORY_MAP[model_class_path]

File: src/ralph/admin/tests/tests_views.py [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            with CaptureQueriesContext(connections['default']) as cqc:
+                change_list = model_admin.changelist_view(self.request)
+                self.assertEqual(query_count, len(cqc))
+
+            change_form = model_admin.changeform_view(
+                self.request, object_id=None
+            )
+            self.assertEqual(change_form.status_code, 200)
+
+
```
Comment (mkurek) [inline src/ralph/admin/tests/tests_views.py]: to many new lines

File: src/ralph/admin/tests/tests_views.py [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    'ralph.operations.models.Problem': 'ralph.operations.tests.factories.ProblemFactory',  # noqa
+    'ralph.reports.models.Report': 'ralph.reports.factories.ReportFactory',
+    'ralph.reports.models.ReportLanguage': 'ralph.reports.factories.ReportLanguageFactory',  # noqa
+    'ralph.supports.models.Support': 'ralph.supports.tests.factories.SupportFactory',  # noqa
+    'ralph.supports.models.SupportType': 'ralph.supports.tests.factories.SupportTypeFactory',  # noqa
+    'ralph.virtual.models.CloudFlavor': 'ralph.virtual.tests.factories.CloudFlavorFactory',  # noqa
+    'ralph.virtual.models.CloudHost': 'ralph.virtual.tests.factories.CloudHostFactory',  # noqa
+    'ralph.virtual.models.CloudProject': 'ralph.virtual.tests.factories.CloudProjectFactory',  # noqa
+    'ralph.virtual.models.CloudProvider': 'ralph.virtual.tests.factories.CloudProviderFactory',  # noqa
+    'ralph.virtual.models.VirtualServer': 'ralph.virtual.tests.factories.VirtualServerFactory',  # noqa
```
Comment (mkurek) [inline src/ralph/admin/tests/tests_views.py]: use `VirtualServerFullFactory`

File: src/ralph/admin/tests/tests_views.py [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+    'ralph.back_office.models.BackOfficeAsset': 'ralph.back_office.tests.factories.BackOfficeAssetFactory',  # noqa
+    'ralph.back_office.models.OfficeInfrastructure': 'ralph.back_office.tests.factories.OfficeInfrastructureFactory',  # noqa
+    'ralph.back_office.models.Warehouse': 'ralph.back_office.tests.factories.WarehouseFactory',  # noqa
+    'ralph.dashboards.models.Dashboard': 'ralph.dashboards.tests.factories.DashboardFactory',  # noqa
+    'ralph.dashboards.models.Graph': 'ralph.dashboards.tests.factories.GraphFactory',  # noqa
+    'ralph.data_center.models.components.DiskShare': 'ralph.data_center.tests.factories.DiskShareFactory',  # noqa
+    'ralph.data_center.models.components.DiskShareMount': 'ralph.data_center.tests.factories.DiskShareMountFactory',  # noqa
+    'ralph.data_center.models.physical.Accessory': 'ralph.data_center.tests.factories.AccessoryFactory',  # noqa
+    'ralph.data_center.models.physical.DataCenter': 'ralph.data_center.tests.factories.DataCenterFactory',  # noqa
+    'ralph.data_center.models.physical.DataCenterAsset': 'ralph.data_center.tests.factories.DataCenterAssetFactory',  # noqa
```
Comment (mkurek) [inline src/ralph/admin/tests/tests_views.py]: use `DataCenterAssetFullFactory`

File: src/ralph/admin/tests/tests_views.py [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+    'ralph.operations.models.Incident': 'ralph.operations.tests.factories.IncidentFactory',  # noqa
+    'ralph.operations.models.Operation': 'ralph.operations.tests.factories.OperationFactory',  # noqa
+    'ralph.operations.models.OperationType': 'ralph.operations.tests.factories.OperationTypeFactory',  # noqa
+    'ralph.operations.models.Problem': 'ralph.operations.tests.factories.ProblemFactory',  # noqa
+    'ralph.reports.models.Report': 'ralph.reports.factories.ReportFactory',
+    'ralph.reports.models.ReportLanguage': 'ralph.reports.factories.ReportLanguageFactory',  # noqa
+    'ralph.supports.models.Support': 'ralph.supports.tests.factories.SupportFactory',  # noqa
+    'ralph.supports.models.SupportType': 'ralph.supports.tests.factories.SupportTypeFactory',  # noqa
+    'ralph.virtual.models.CloudFlavor': 'ralph.virtual.tests.factories.CloudFlavorFactory',  # noqa
+    'ralph.virtual.models.CloudHost': 'ralph.virtual.tests.factories.CloudHostFactory',  # noqa
```
Comment (mkurek) [inline src/ralph/admin/tests/tests_views.py]: use `CloudHostFullFactory`

**Comment by ar4s** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**